content/client-lib-development-guide/features.textile

```diff
@@ -339,6 +340,14 @@ h3(#rest-encryption). Encryption
 h3(#rest-compatibility). Forwards compatibility
 * @(RSF1)@ The library must apply the "robustness principle":https://en.wikipedia.org/wiki/Robustness_principle in its processing of requests and responses with the Ably system. In particular, deserialization of Messages and related types, and associated enums, must be tolerant to unrecognised attributes or enum values. Such unrecognised values must be ignored.
 
+h3(#batch-operations). Batch Operations
+* @(BO1)@ The batch operations functions must use the REST endpoints in Batch Mode, sending a single request containing all specified data
+* @(BO2)@ Batch operations must be able to be performed for the following:
+** @(BO2a)@ @BatchOperations::publish@ publishes messages against one or more channels with one or more messages
+*** @(B02a1)@ Functions should be provided to pass either an array or a single object for both Message and Channel ID, or an array of @BatchSpec@ objects
+** @(BO2b)@ @BatchOperations::getPresence@ retrieves the presence data for one or more channels
+* @(BO3)@ When all passed arrays contain a single object, the underlying request is functionally identical to its non-batch equivalent, but the returned result should be a @BatchResponse@ object.
```

Comment:
```suggestion
* @(BO3)@ When a batch operation only contains one batch, the underlying request is functionally identical to its non-batch equivalent, but the returned result should be a @BatchResponse@ object.
```
An attempt to simplify this wording and use language that's a bit more programming language neutral.